camelcase.go

```diff
@@ -7,28 +7,17 @@ import "unicode"
 // Split splits the camelcase word and returns a list of words. It also
 // supports digits.  Both lower camel case and upper camel case are supported.
 // For more info please check:  http://en.wikipedia.org/wiki/CamelCase
-//
-// Below are some example cases:
-//   lowercase =>       ["lowercase"]
-//   Class =>           ["Class"]
-//   MyClass =>         ["My", "Class"]
-//   MyC =>             ["My", "C"]
-//   HTML =>            ["HTML"]
-//   PDFLoader =>       ["PDF", "Loader"]
-//   AString =>         ["A", "String"]
-//   SimpleXMLParser => ["Simple", "XML", "Parser"]
-//   vimRPCPlugin =>    ["vim", "RPC", "Plugin"]
-//   GL11Version =>     ["GL", "11", "Version"]
-//   99Bottles =>       ["99", "Bottles"]
-//   May5 =>            ["May", "5"]
-//   BFG9000 =>         ["BFG", "9000"]
```

Comment:
Note, I've removed these, as the test is now a go doc example - so in the go docs these examples can still be seen.